entity-framework/core/what-is-new/ef-core-2.1.md

```diff
@@ -76,7 +76,7 @@ An EF Core model can now include query types. Unlike entity types, query types d
 - Mapping to queries defined in the model
 - Serving as the return type for `FromSql()` queries
 
-Read the [section on query types](xref:core/modeling/query-types) for more information about this topic.
+Read the [section on query types](xref:core/modeling/keyless-entity-types) for more information about this topic.
```

Comment:
Is keeping link title as query types here intentional?

Reply:
Yes, in 2.1 they were query types